src/test/java/org/jenkinsci/plugins/envinject/sevice/PropertiesLoaderTest.java

```diff
@@ -148,7 +148,7 @@ public void contentWithVarsToResolve() throws Exception {
     }
 
     private void checkWithVarsToResolve(boolean fromFile) throws Exception {
-        String content = "KEY1 =${VAR1_TO_RESOLVE}\nKEY2=https\\://github.com\nKEY3=${VAR3_TO_RESOLVE}\\\\otherContent";
+        String content = "KEY1 =${VAR1_TO_RESOLVE}\nKEY2=https://github.com\nKEY3=${VAR3_TO_RESOLVE}\\\\otherContent";
```

Comment:
This change makes me think that there are some implied regressions here

Reply:
I misread this the first time. Why would you expect a variable `KEY2` which consists of `https:\\github.com` to resolve to `https://github.com`?

Those would be 2 different slash values and I would not expect that behavior. The test fails even without my changes

Reply:
@alexanderrtaylor I think @oleg-nenashev is concerned of the backwards compatibility

Reply:
Let me know if you can't work on this I'm blocked from using the latest plugin because of this very issue

Reply:
I am taking a look at your suggestion now and will try it out locally